src/language/grammar.tx

```diff
@@ -4,18 +4,27 @@ Specification:
 	constraints += Constraint*
 ;
 
-Param: key = ID ':' value = STRING ;
+Param: 'Param' key = ID ':' value = STRING ;
 
 Actor: Car | Pedestrian ;
 
 Car:
 	'Car' name = ID
+	isEgo ?= 'isEgo'
```

Comment:
This basically means if the user writes `isEgo` here, we will set an attribute `isEgo` to `True` (so we don't need to do something like `isEgo: True`)